src/scripts/components/component.es6

```diff
@@ -96,7 +96,7 @@ module.exports = class Component {
 	destroy() {
 		this.detachListeners()
 
-		for (let prop in this) {
+		for (const prop in this) {
```

Comment:
Podle mě jednoduše není správně nad tím přemýšlet stylem „vidím `for`, dám tam `let`“ (až budem používat `for of`, taky budem psát spíš `const`), protože to zbytečně ignoruje, jak to celé funguje. Podle téhle logiky bychom psali `let` úplně všude, protože _občas_ tu proměnnou člověk holt mění, a to je podle mě nesprávné.

Navíc v té smyčce se reference právě implicitně nemění, ale vytváří se pro každou iteraci nová, to je celý důvod vzniku `let` a `const`. Viz např. klasika:
```
var foo = {
  a: '',
  b: '',
  c: '',
}

for (var key in foo) {
  setTimeout(function () {
    console.log(key)
  }, 0)
}
```
Myslím, že ve `for`ech úplně v pohodě může být `const` a že to není nijak matoucí, obzvlášť, když si člověk uvědomí, že `for (let i = 0; i < foo.length; i++)` je vlastně jen syntactic sugar pro
```
{
    let i = 0;
    while (i < foo.length) {
        // …
        i += 1;
    }
}
```
Pak je jasné, proč `const` nefunguje a není to vůbec nečitelné.